src/main/java/com/khoros/twitterapp/resources/HomeFeedResource.java

```diff
@@ -33,11 +37,37 @@ public HomeFeedResource(Twitter factory) {
 
     @GET
     public Response get() {
+
+        logger.info("Attempting to retrieve Twitter Home timeline.");
+        logger.debug("Configuration setup: {}", conf);
+
         try {
+
             List<Status> tweetsFeed = factory.getHomeTimeline();
             return Response.status(HttpURLConnection.HTTP_OK).entity(tweetsFeed).build();
+
         } catch (TwitterException feedException) {
+
+            logger.info("Timeline retrieval aborted. Twitter Exception thrown." );
+
+            if (feedException.isCausedByNetworkIssue()) {
+
+                logger.debug("Exception Caused By Network Issues: {}", feedException.isCausedByNetworkIssue());
+
+            } else if (feedException.exceededRateLimitation()) {
+
+                logger.debug("Current Rate Limit: {}", feedException.getRateLimitStatus());
+                logger.debug("Exceed Rate Limitation: {}", feedException.exceededRateLimitation());
+
+            } else {
+
+                logger.debug("Twitter API Access Level: {}", feedException.getAccessLevel());
+                logger.debug("TwitterException Error Message: {}", feedException.getErrorMessage());
```

Comment:
Are these logs the correct logging level? In a similar vein, are they all necessary?